spec/createsend_rails/deliverer_spec.rb

```diff
@@ -24,6 +31,26 @@
       end
     end
 
+
+    context 'sailthru delivery' do
+      let(:request) { { api: 'abcdef', delivery_system: 'sailthru' } }
+      let(:raw_source) { '{ "smart_email_id": "example-id-555", "delivery_system": "sailthru" }' }
+      let(:message) { double ( { body: { raw_source: { delivery_system: 'sailthru' } } } ) }
+
+      it 'return successfully' do
+        sailthru_mock_objct = double
+        expect_any_instance_of(described_class).to receive(:sailthru_client).and_return(sailthru_mock_objct)
+        expect(sailthru_mock_objct).to receive(:api_post).and_return(true)
```

Comment:
I would assert the parameters, i.e. `expect(...).to receive(:api_post).with(...).and_return(...)`